src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,94 @@
+.product-card {
+  box-sizing: border-box;
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  font-family: "Roboto", sans-serif;
+
+  background-color: $background-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    display: block;
+    height: 134px;
+    margin: auto;
+    margin-bottom: 40px;
+    object-position: center;
+  }
+
+  &__product-name {
+    margin-bottom: 4px;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $text-color-dark;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $text-color-light;
+  }
+
+  &__rate {
+    display: flex;
+    margin-bottom: 24px;
+    justify-content: space-between;
+    align-items: end;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    font-weight: 400;
+    color: $text-color-dark;
+  }
+
+  &__price-container {
+    display: flex;
+    margin-bottom: 16px;
+    justify-content: space-between;
+  }
+
+  &__price {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: $text-color-light;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $text-color-dark;
+  }
```

Comment:
Suggestion: You can fold this one in
`&__price{
    &-container{
`

Reply:
But they are different elements of one block - product-card. They just have similar names. And I think we shouldn't put one element in another.